Produlytics/ui/back-end/src/main/java/it/deltax/produlytics/uibackend/repositories/CaratteristicaRepository.java

```diff
@@ -0,0 +1,13 @@
+package it.deltax.produlytics.uibackend.repositories;
+
+import it.deltax.produlytics.persistence.CharacteristicEntity;
+import it.deltax.produlytics.persistence.CharacteristicEntityId;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface CaratteristicaRepository extends CrudRepository<CharacteristicEntity, CharacteristicEntityId> {
```

Comment:
Le repository dovrebbero cambiare nome in inglese.

Reply:
Hanno detto che lo faranno loro

Reply:
Sì, ma che refactoring è allora?